litmus-portal/frontend/src/views/CreateWorkflow/ChooseAWorkflowAgent/index.tsx

```diff
@@ -217,28 +217,34 @@ const ChooseWorkflowAgent = forwardRef((_, ref) => {
         {/* Cluster Data */}
         <div className={classes.agentWrapperDiv} data-cy="AgentsRadioGroup">
           {filteredCluster.map((cluster) => (
-            <LitmusCard
-              key={cluster.cluster_id}
-              glow={currentlySelectedAgent === cluster.cluster_id}
-              width="100%"
-              height="4rem"
-              className={classes.litmusCard}
-              borderColor={
-                currentlySelectedAgent === cluster.cluster_id
-                  ? palette.primary.main
-                  : palette.border.main
-              }
+            <RadioGroup
```

Comment:
@amityt With this approach, This RadioGroup will have only one button under itself.
I think the RadioGroup Component should be out of the map()?